### PR TITLE
Add gzip compression for bootstrap.min.css

### PR DESCRIPTION
**Pull request by christodenny**
Serve a locally gzip compressed version of bootstrap.min.css. Still have to encode it as text/css so browser knows to use it.

Didn't see any other way to serve a gzip version of this with an http.FileServer object without introducing a compression wrapper a la https://gist.github.com/bryfry/09a650eb8aac0fb76c24. This also works but it does unnecessary computation to compress a static file every time.

Yes I know this is the sort of thing that should be in a CDN ¯\_(ツ)_/¯

Also add gitignore for vscode workspace file and locally built lastwin binary.

### CODE REVIEW

**Review by williamvwang — APPROVED** [review]
LG(zip)TM

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+lastwin.code-workspace
```
Comment (rjchee) [inline .gitignore:2]: why should your IDE-specific files go in our repo smh

File: main.go [hunk 2]
```diff
@@ -60,6 +60,7 @@ func lastWinHandler(w http.ResponseWriter, r *http.Request) {
 	team := allTeams[teamname]
 	espnLink := fmt.Sprintf(urls[team.Division]["espn"], team.ID)
 	imgLink := fmt.Sprintf(urls[team.Division]["img"], team.ID)
+	// TODO: do smth about thundering herd with this cache
```
Comment (rjchee) [inline main.go:63]: what thundering herd lmao

**Review by rjchee — COMMENTED** [review]
why should your IDE-specific files be in our sacred gitignore?

**Pull request merged**